packages/ui/src/components/Select/select.tsx

```diff
@@ -66,8 +66,7 @@ const Select = <ValueType extends SelectValueTypeBase>(props: SelectProps<ValueT
         }
 
         if (title) {
-            // TODO(DOMA-5597): Move it to change event; Also key should be used as well
```

Comment:
left a todo about key and multiple select options please, it's still not fixed :)

Reply:
Done